root_files/default-urls.json

```diff
@@ -0,0 +1 @@
/", "/en-US/foundation/moco/", "/en-US/foundation/openwebfund/more/", "/en-US/foundation/trademarks/", "/en-US/foundation/trademarks/community-edition-permitted-changes/", "/en-US/foundation/trademarks/community-edition-policy/", "/en-US/foundation/trademarks/distribution-policy/", "/en-US/foundation/trademarks/faq/", "/en-US/foundation/trademarks/l10n-website-policy/", "/en-US/foundation/trademarks/list/", "/en-US/foundation/trademarks/policy/", "/en-US/foundation/trademarks/poweredby/faq/", "/en-US/grants/", "/en-US/grants/reports/gnome-haeger-report/", "/en-US/grants/reports/ushahidi-chile-report/", "/en-US/internet-health/", "/en-US/internet-health/decentralization/", "/en-US/internet-health/digital-inclusion/", "/en-US/internet-health/open-innovation/", "/en-US/internet-health/privacy-security/", "/en-US/internet-health/web-literacy/", "/en-US/mission/", "/en-US/moss/", "/en-US/moss/foundational-technology/", "/en-US/moss/mission-partners-india/", "/en-US/moss/mission-partners/", "/en-US/moss/secure-open-source/", "/en-US/newsletter/", "/en-US/newsletter/country/success/", "/en-US/newsletter/developer/", "/en-US/newsletter/firefox/", "/en-US/newsletter/mozilla/", "/en-US/newsletter/opt-out-confirmation/", "/en-US/plugincheck/", "/en-US/press/press-inquiry/", "/en-US/press/speakerrequest/", "/en-US/privacy/", "/en-US/privacy/archive/", "/en-US/privacy/archive/firefox/2006-10/", "/en-US/privacy/archive/firefox/2008-06/", "/en-US/privacy/archive/firefox/2009-01/", "/en-US/privacy/archive/firefox/2009-09/", "/en-US/privacy/archive/firefox/2010-01/", "/en-US/privacy/archive/firefox/2010-12/", "/en-US/privacy/archive/firefox/2011-06/", "/en-US/privacy/archive/firefox/2012-06/", "/en-US/privacy/archive/firefox/2012-09/", "/en-US/privacy/archive/firefox/2012-12/", "/en-US/privacy/archive/firefox/2013-05/", "/en-US/privacy/archive/firefox/third-party/", "/en-US/privacy/archive/hello/2014-11/", "/en-US/privacy/archive/hello/2016-03/", "/en-US/privacy/archive/persona/2017-07/", "/en-US/privacy/archive/thunderbird/2010-06/", "/en-US/privacy/archive/websites/2013-08/", "/en-US/privacy/facebook/", "/en-US/privacy/firefox-cliqz/", "/en-US/privacy/firefox-focus/", "/en-US/privacy/firefox-klar/", "/en-US/privacy/firefox-os/", "/en-US/privacy/firefox-rocket/", "/en-US/privacy/firefox/", "/en-US/privacy/principles/", "/en-US/privacy/thunderbird/", "/en-US/privacy/websites/", "/en-US/projects/calendar/", "/en-US/security/", "/en-US/security/advisories/", "/en-US/security/advisories/cve-feed.json", "/en-US/security/bug-bounty/", "/en-US/security/bug-bounty/faq-webapp/", "/en-US/security/bug-bounty/faq/", "/en-US/security/bug-bounty/hall-of-fame/", "/en-US/security/bug-bounty/web-eligible-sites/", "/en-US/security/bug-bounty/web-hall-of-fame/", "/en-US/security/client-bug-bounty/", "/en-US/security/known-vulnerabilities/", "/en-US/security/known-vulnerabilities/older-vulnerabilities/", "/en-US/security/web-bug-bounty/", "/en-US/styleguide/", "/en-US/technology/"]
```

Comment:
I think this file and sitemap.xml were added accidentally. Looks like .gitignore was updated in https://github.com/mozilla/bedrock/commit/87233d2b30d776f1a54f18ed8819c3da64529680 so I think these shouldn't be in this PR. A rebase might fix it.

Reply:
Unintended Files In My Pull Requests are my nemesis!